provider/cmd/pulumi-resource-azure/bridge-metadata.json

```diff
@@ -13153,25 +13161,6 @@
                 "current": "azure:marketplace/roleAssignment:RoleAssignment",
                 "majorVersion": 6
             },
-            "azurerm_media_services_account_filter": {
```

Comment:
gone?

Reply:
This was meant to be removed in the major version, but was accidentally not deleted: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/4.0-upgrade-guide#azurerm_media_services_account_filter